Frontend/ui-customer/index.js

```diff
@@ -19,7 +19,8 @@ function loadConfig() {
 }
 
 window.session_id = guid();
-window.version = __VERSION__;
+window.version = window.version || {};
+window.version["app"] = __VERSION__;
```

Comment:
how is this changed? would it affect the the `getAppVersion` function? which gives the app version in PROD

Reply:
No. It won't affect in PROD. It will work fine. Fixed the bundleVersion function.